src/foam/nanos/auth/token/TokenExpiredException.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  name: 'TokenExpiredException',
+  package: 'foam.nanos.auth.token',
+  extends: 'foam.nanos.auth.AuthenticationException',
+  javaGenerateDefaultConstructor: false,
+  javaGenerateConvenienceConstructor: false,
+
+  properties: [
+    {
+      name: 'exceptionMessage',
+      value: 'Token has expired'
```

Comment:
This message isn't needed. The foam exception system will parse the exception name into message:  Token Expired
and do translation on this if required.

Reply:
These exceptions are handled in Controller.js and use the message to determine error 
e.g.
 `           if ( e.message && e.message === "Token has expired" ) {`
Is there another way to do this on client side without exceptionMessage property?

Reply:
AND if i remove the exceptionMessage property e.message is empty

Reply:
Recommend you use id which is (cls_.id) and in this case becomes: "TokenExpiredException".